docs/xmlprofiles.rst

```diff
@@ -660,6 +667,39 @@ List with the possible configuration parameter:
 | ``<offsetd3>``          | Unicast user data offset.   | ``uint16`` | 11      |
 +-------------------------+-----------------------------+------------+---------+
 
+.. _ParticipantAllocationType:
+
+Participant allocation parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section of the :class:`Participant's rtps` configuration allows defining parameters related with allocation
+behavior on the participant.
+
+.. literalinclude:: ../code/XMLTester.xml
+    :language: xml
+    :start-after: <!-->XML-PARTICIPANT-ALLOCATION<-->
+    :end-before: <!--><-->
+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - Name
+     - Description
+     - Values
+     - Default
+   * - ``<max_unicast_locators>``
+     - Maximum number of unicast locators expected on a remote entity.
+       It is recommended to use the maximum number of network interfaces found on any machine the
+       participant will connect to.
+     - ``UInt32``
+     - 4
```

Comment:
Just out of interest, why is the default value 4?

Reply:
We thought it is a good compromise value. It is not usual to have more than 4 network interfaces.